test/acceptance/server-ha-enterprise.bats

```diff
@@ -132,15 +129,15 @@ load _helpers
 
 # setup a consul env
 setup() {
-  helm install https://github.com/hashicorp/consul-helm/archive/v0.8.1.tar.gz \
-    --name consul \
+
+  helm install consul https://github.com/hashicorp/consul-helm/archive/v0.8.1.tar.gz \
     --set 'ui.enabled=false' \
+    --set server.affinity=null \
```

Comment:
To your question, I think the acceptance tests have usually been run against a cluster in GKE, created with the terraform in `test/terraform/`. [kind](https://github.com/kubernetes-sigs/kind) is another option for testing locally, since it supports multi-node k8s clusters.

For example, with this config yaml I can create a cluster locally with 3 worker nodes and run the tests, albeit a little slowly:

```yaml
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
- role: worker
- role: worker
- role: worker
```

```shell
kind create cluster --image kindest/node:v1.15.7 --config ./kind-4node.yaml
```

Reply:
`kind` is also being used for the CSI unit testing.

Reply:
Thank you for that tip! I installed `kind` locally and have now edited the acceptance tests so they're fully passing locally.